canonical/store-components
# Add line clamp to package card description

## Done
Added a line clamp limit to package card descriptions to avoid overly long card heights.

## QA
- Go to https://store-components-56.demos.haus/?path=/story/defaultcard--long-description
- Check that the description is cut off after 5 lines
